styles/fonts.css

```diff
@@ -0,0 +1,45 @@
+/* montserrat-300 - latin */
```

Comment:
Are these comments above the `@font-face` at-rule declarations really needed? It's visible when looking at the `src` rule